client/allocrunner/network_hook_test.go

```diff
@@ -28,17 +28,22 @@ func (m *mockNetworkIsolationSetter) SetNetworkIsolation(spec *drivers.NetworkIs
 	test.Eq(m.t, m.expectedSpec, spec)
 }
 
-type mockNetworkStatusSetter struct {
+type mockNetworkStatus struct {
 	t              *testing.T
 	expectedStatus *structs.AllocNetworkStatus
-	called         bool
+	calls          int
 }
 
-func (m *mockNetworkStatusSetter) SetNetworkStatus(status *structs.AllocNetworkStatus) {
-	m.called = true
+func (m *mockNetworkStatus) SetNetworkStatus(status *structs.AllocNetworkStatus) {
+	m.calls++
 	test.Eq(m.t, m.expectedStatus, status)
 }
 
+func (m *mockNetworkStatus) NetworkStatus() *structs.AllocNetworkStatus {
+	m.calls++
```

Comment:
in general, I like to distinguish between the different methods when counting, to remove ambiguity in test assertions.  e.g. `getCalls` and `setCalls`